common/shared/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>common</artifactId>
+    <groupId>org.opencwa</groupId>
+    <version>${revision}</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>shared</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.opencwa</groupId>
+      <artifactId>protocols</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
I think a lot of these dependencies are not needed inside this project. Maybe we can clean this pom.xml up a little bit

Reply:
Removed them.